lib/awesome_spawn.rb

```diff
@@ -126,7 +126,7 @@ def launch(env, command, spawn_options)
     else
       output, error, status = Open3.capture3(env, command, spawn_options)
     end
-    return output, error, status && status.exitstatus
+    return output, error, status.pid, status && status.exitstatus
```

Comment:
Looks like there is a check for status here for some reason. It would be strange to have `status.pid` unconditionally, but then check for if status is falsey in the next return value

Reply:
Maybe return the entire status object and handle these conditions in the caller?